Scenarios/Radial Control/RadialControlSample/RadialControlSample/ViewController.m

```diff
@@ -0,0 +1,169 @@
+    
+    
+    [self.view addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"H:|-[title]-|"
+                                                                      options:0
+                                                                      metrics:metrics
+                                                                        views:views]];
+    [self.view addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"H:|-[info]-|"
+                                                                      options:0
+                                                                      metrics:metrics
+                                                                        views:views]];
+    [self.view addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"H:|-[slider]-|"
+                                                                      options:0
+                                                                      metrics:metrics
+                                                                        views:views]];
+    [self.view addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"H:|-[sliderLabel]-|"
+                                                                      options:0
+                                                                      metrics:metrics
+                                                                        views:views]];
+    [self.view addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"H:|-[switch]-|"
+                                                                      options:0
+                                                                      metrics:metrics
+                                                                        views:views]];
+    [self.view addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"V:|-pad-[title]-[info(demoInfoHeight)]-margin-[slider(sliderHeight)]-[sliderLabel]-margin-[switch(switchHeight)]"
+                                                                      options:0
+                                                                      metrics:metrics
+                                                                        views:views]];
+    
+#ifdef WINOBJC
+    [[UIApplication sharedApplication] setStatusBarHidden:YES]; // Deprecated in iOS
+    
+    // Create a reference to the radial controller
+    self.radialController = [WUIRadialController createForCurrentView];
+
+    // Get the radial controller menu
+    WUIRadialControllerMenu* menu = self.radialController.menu;
+
+    // Get the menu items
+    NSMutableArray* menuItems = menu.items;
+
+    // Create a new menu item
+    WUIRadialControllerMenuItem* newMenuItem = [WUIRadialControllerMenuItem createFromKnownIcon:@"Custom Tool" value:WUIRadialControllerMenuKnownIconRuler];
+
+    // Add a new menu item
+    [menuItems addObject:newMenuItem];
+	
+    __block ViewController* blockSelf = self; // Ensures self will not be retained
```

Comment:
Agreed, the comment is for __weak and not __block. I'm going to change the code to use __weak instead. Does that cover your concern?